crates/bevy_ui/src/lib.rs

```diff
@@ -47,10 +50,27 @@ pub enum UiSystem {
     Focus,
 }
 
+/// The current scale of the UI.
+///
+/// A multiplier to fixed-sized ui values.
+/// **Note:** This will not affect non-fixed ui values like [`Val::Percent`]
```

Comment:
```suggestion
/// **Note:** This will only affect fixed ui values like [`Val::Px`]
```
I think it's clearer to say the other way around, it will _only_ affect `Val::Px`.

Reply:
Yup that is better, thanks :)